src/components/CharClassBox.js

```diff
@@ -0,0 +1,26 @@
+import '../css/CharClassBox.css';
+import { Fragment } from 'react';
+import CharClass from './CharClass';
+import useFetch from '../hooks/useFetch';
+import endpoints from '../endpoints.js';
+
+const CharClassBox = () => {
+  const {data, isLoading, error} = useFetch(endpoints.classes);
+  const classes = data?.results;
+
+  return (
+    <div className="char-class-box">
+      {isLoading ? <>loading...</> :
+        <>
+          {error ? <>{error}</> :
+            <>
+              {classes.map(charClass => <CharClass charClass={charClass} key={charClass.index}/>)}
```

Comment:
Thanks for adding the key @augustreid!  I think that it's really strange that it didn't get mad at me forgetting it... and I was counting on React to give me a warning, cause I always forget the key.

Reply:
Yeah, strange! on mine it was throwing an error but still rendering. At least it's an easy fix.